lib/parse.js

```diff
@@ -1320,11 +1320,17 @@ function parse($TEXT, options) {
             var type = start.type;
             var name = as_property_name();
             if (type == "name" && !is("punc", ":")) {
+                var createAccessor = function() {
```

Comment:
Tried that (a thought: "Silly me. off course I have to return in JS, this is not Scala "), but I still get the parsing error:

> 'SyntaxError: Unexpected token punc «(», expected punc «,»'

My code was:
                var createAccessor = function() {embed_tokens(function() {
                    return function_(AST_Accessor);
                })};